tsconfig.json

```diff
@@ -1,10 +1,11 @@
 {
   "extends": "@foxglove/tsconfig/base",
-  "include": ["./src/**/*"],
+  "include": ["./src/**/*", "*.d.ts"],
```

Comment:
Including files from the root directory might mess up the contents of the dist directory? (e.g. by creating `dist/esm/src`). I haven't checked on your branch but this happened to me in the past, so worth double checking.

If that is an issue, I don't think there is any real need to have our tsconfig cover these one liner `.d.ts` files (we don't in other packages).

Reply:
It did not impact the build. I tested it with `yarn pack` and install in an empty js project.